index.html

```diff
@@ -115,71 +114,71 @@ <h1 class="blue">Top Restaurant Chains</h1>
+                    <span class="Cities">Kolkata</span>
+                    <span class="Cities">Maharashtra</span>
+                    <span class="Cities">Karnataka</span>
+                    <span class="Cities">Kochi</span>
                 </div>
                 <div class="menu">
 
-                    <span>Kerela</span>
-                    <span>Agra</span>
-                    <span>Leh</span>
-                    <span>Kota</span>
-                    <span>Patan</span>
+                    <span class="Cities">Kerela</span>
+                    <span class="Cities">Agra</span>
+                    <span class="Cities">Leh</span>
+                    <span class="Cities">Kota</span>
+                    <span class="Cities">Patan</span>
                 </div>
 
                 <div class="menu">
-                    <span>Maharashtra</span>
-                    <span>Karnataka</span>
-                    <span>Kochi</span>
-                    <span>Goa</span>
-                    <span>Kolkata</span>
+                    <span class="Cities">Maharashtra</span>
+                    <span class="Cities">Karnataka</span>
+                    <span class="Cities">Kochi</span>
+                    <span class="Cities">Goa</span>
+                    <span class="Cities">Kolkata</span>
                 </div>
 
                 <div class="menu">
-                    <span>Kerela</span>
-                    <span>Agra</span>
-                    <span>Leh</span>
-                    <span>Kota</span>
-                    <span>Patan</span>
+                    <span class="Cities">Kerela</span>
+                    <span class="Cities">Agra</span>
+                    <span class="Cities">Leh</span>
+                    <span class="Cities">Kota</span>
+                    <span class="Cities">Patan</span>
                 </div>
 
                 <div class="menu ll dn">
-                    <span>Maharashtra</span>
-                    <span>Karnataka</span>
-                    <span>Kochi</span>
-                    <span>Goa</span>
-                    <span>Kolkata</span>
+                    <span class="Cities">Maharashtra</span>
+                    <span class="Cities">Karnataka</span>
+                    <span class="Cities">Kochi</span>
+                    <span class="Cities">Goa</span>
+                    <span class="Cities">Kolkata</span>
                 </div>
                 <div class="menu ll dn">
 
-                    <span>Goa</span>
-                    <span>Kolkata</span>
-                    <span>Maharashtra</span>
-                    <span>Karnataka</span>
-                    <span>Kochi</span>
+                    <span class="Cities">Goa</span>
+                    <span class="Cities">Kolkata</span>
+                    <span class="Cities">Maharashtra</span>
+                    <span class="Cities">Karnataka</span>
+                    <span class="Cities">Kochi</span>
```

Comment:
Isn't it way easier to use the span element instead of adding classes everywhere?
```css
.menu span:hover {
 color: #ffc04d;
 box-shadow: var(--orange) 0px 8px 8px;
}
```

The same for "Restaurants"